lnrpc/routerrpc/router_server.go

```diff
@@ -485,35 +485,6 @@ func (s *Server) trackPayment(paymentHash lntypes.Hash,
 			status.State = state
 		}
 
-		// Extract the last route from the given list of HTLCs. This
-		// will populate the legacy route field for backwards
-		// compatibility.
-		//
-		// NOTE: For now there will be at most one HTLC, this code
```

Comment:
would imagine this should be part of a different pr, no?

Reply:
This PR is the moment we make `routerrpc` official. I don't think we should make a field official that is non-sensical.

Reply:
okay, i buy that

Reply:
(i know this is done deal but i still agree with the different PR)